deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml

```
@@ -68,6 +68,14 @@ objects:
     verbs:
     - create
     - delete
+  - apiGroups:
+    - results.tekton.dev
+    resources:
+    - results
+    - records
+    verbs:
+    - get
+    - list
```

I guess you want `watch` too?
```suggestion
    - list
    - watch
```

---

Not needed, it's not used in their clusterrole -> https://github.com/tektoncd/results/blob/main/config/default-clusterroles.yaml

---

This is not a real resource, but it's used by tekton results api to use RBAC. https://github.com/tektoncd/results/tree/main/docs/api